draft-ietf-quic-transport.md

```diff
@@ -1461,127 +1462,215 @@ connection if the integrity check fails with a PROTOCOL_VIOLATION error code.
-port mappings.
-
-In order to make it more difficult for passive eavesdroppers to track a
-connection as it migrates, a client can use different connection IDs when
-sending from different addresses. The server can send NEW_CONNECTION_ID
-frames to provide the client with additional connection IDs to be used during
-migration ({{migration-linkability}}).
-
-Any time a new remote endpoint is detected, such as when a server
-receives a packet for a known connection ID from a different address, the remote
-endpoint MUST be validated ({{migration-validate}}).
-
-Any time a new local endpoint is used, such as when a client chooses
-to migrate a connection onto a different network interface, the path can be
-probed to check reachability ({{migration-probe}}). An endpoint may choose to
-probe a network path before migrating the connection, or it may choose to
-migrate the connection immediately. For example, a client that detects that its
-current network path is degraded may choose to probe alternate paths and only
-migrate if an alternate path is preferred over the current one. A client may
-also find that a path suddenly becomes inoperable, such as when a user
-unplugs a cable or other network state changes, and may elect to migrate
-without probing since it has no alternative if it wishes to maintain the
+QUIC connections are identified by 64-bit Connection IDs. QUIC's connection IDs
+allow connections to survive changes to the client's address (client's IP
+address and/or port), such as those caused by a client migrating to a new
+network or a NAT rebinding changing the client's public address.
+
+A client migrating to a new network SHOULD use a new connection ID for packets
+sent from the new address if at all possible, as discussed in
+{{migration-linkability}}.  A client may be unaware of connection migration due
+to NAT rebinding and may consequently not change its connection ID on NAT
+rebindings.
+
+A server that receives a packet for a known connection ID from a new client
+address SHOULD validate this new address, as discussed in
+{{migration-validate}},
+
+
+### Path Validation and PMTU Verification
+
+PATH_CHALLENGE ({{frame-path-challenge}}) and PATH_RESPONSE
+({{frame-path-response}}) frames MAY be used for validating that an endpoint
+owns an address, or to verify the PMTU to/from an address.
+
+A server validates a client's address by sending a PATH_CHALLENGE frame
+containing a payload that is hard to guess to that address.  The server
+considers the address to be valid when a PATH_RESPONSE frame containing the same
+payload is received from the client.
+
+Similarly, a client verifies the PMTU of a network path from a new client
+address to the server by sending a PMTU-sized packet carrying a PATH_CHALLENGE
+frame. The client considers the PMTU verified when a PATH_RESPONSE frame
+containing the same payload sent by the server to the new client address is
+received.
```

Comment:
Well, the client doesn't know the MTU on the reverse path, so it can't detect whether it's PMTU sized or not... besides, this is not a security concern, so the client doesn't need to do anything special, IMO. I've added some more text below in server behavior.

Reply:
Ah, fair enough.